docs/docs/configuration/filtering.md

```diff
@@ -215,6 +215,12 @@ generic-unwanted-rule
 aws-account-id
 ```
 
+For a container image, such as the one used below, follow these steps to add a vulnerability to the `trivyignore` file: 
```

Comment:
What kind of comments did we get? It just adds CVE-ID to `.trivyignore`. Why do we want to navigate users to AVD?

Reply:
Well people were pointing out that the steps are not documented -- so without documentation I cannot know what is wrong with the existing documentation -- if you think this is unnecessary, then we can leave it

Reply:
I'm wondering if they wanted to know how to ignore misconfiguration findings. Ignoring vulnerabilities is too simple. When CVE-2019-1563 is detected, and they want to ignore it, they just add it to `.trivyignore`. There are no special steps to explain.

We should clarify what is unclear for people in our documentation.